docs/11.0/guides/sidebar.js

```diff
@@ -167,6 +167,7 @@ const upgradeAndMigrationItems = [
   'guides/upgrade-and-migration/migrating-from-7.4-to-8.0',
   'guides/upgrade-and-migration/migrating-from-8.4-to-9.0',
   'guides/upgrade-and-migration/migrating-from-9.0-to-10.0',
+  'guides/upgrade-and-migration/migrating-from-10-to-11',
```

Comment:
All migration guides links are using `9.0`, `10.0` notation. Do we intentionally drop this notation form now on?

```suggestion
  'guides/upgrade-and-migration/migrating-from-10.0-to-11.0',
```

Reply:
I was thinking of simplifying the naming, but you're right, it's better to stick with the same convention, done: 9513463 2a732f2

Reply:
Sure, we can do that. If you see value in simplification then previous migration guides can be renamed and fixed with a 301 redirect. We would have to coordinate this change with Budnix and Chris first

Both commits fix only `next` dir, we should apply same changes to `v11`

Reply:
@wojciechczerniak afa6b0c